src/config.ts

```diff
@@ -1 +1,9 @@
 export const COUNTLY_API_URL = 'https://countly.ipfs.tech'
+export const COUNTLY_SETUP_DEFAULTS = {
+  url: COUNTLY_API_URL,
+  autoTrack: true,
+  interval: 5000,
+  max_events: 1000,
+  queue_size: 1000,
```

Comment:
Do we know how `max_events` and `queue_size` relate? Does this mean that if there's a ton of data, our countly server will be getting `max_events` * `queue_size` * (data_size) of data in a single request?

Reply:
queue_size usage: https://github.com/Countly/countly-sdk-web/blob/1c72a3b5d4c1e66031e3ecc5779adfe030ab21e0/lib/countly.js#L3586-L3588
max_events: https://github.com/Countly/countly-sdk-web/blob/1c72a3b5d4c1e66031e3ecc5779adfe030ab21e0/lib/countly.js#L3655-L3666

Reply:
it looks like if it's greater than request queue, they're just dropped :/